linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml

```
@@ -56,3 +56,5 @@ template:
     vars:
         variable: maxclassrepeat
         operation: less than or equal
+        nonzero_check: "false"
+        nonzero_check@ol7: "true"
```

```suggestion
        nonzero_check: "true"
```

I believe this change can be applied to all products. RHEL STIG also brings this change in the latest releases. Please update the description accordingly as well.